.github/BOTMETA.yml

```diff
@@ -1036,7 +1036,7 @@ files:
   $modules/read_csv.py:
     maintainers: dagwieers
   $modules/redfish_:
-    ignore: jose-delarosa
+    ignore: jose-delarosa varini-hp prativa-n Gayathirideviramasamy TSKushal
```

Comment:
Why is this needed?

Reply:
Hi, 
This was raising an error in one if the tests when we tried to do it without adding this before.

Reply:
Hope this is clarified now. Can I resolve it?